plugins/azure/virtualmachines/vmSecurityType.js

```diff
@@ -34,10 +43,10 @@ module.exports = {
             }
 
             virtualMachines.data.forEach(virtualMachine => {
-                if (virtualMachine.securityProfile && virtualMachine.securityProfile.securityType == 'TrustedLaunch') {
-                    helpers.addResult(results, 0, 'Trusted launch is selected as security type for virtual machine', location, virtualMachine.id);
+                if (virtualMachine.securityProfile && virtualMachine.securityProfile.securityType.toLowerCase() == config) {
```

Comment:
```suggestion
                if (virtualMachine.securityProfile && virtualMachine.securityProfile.securityType.toLowerCase() == config) {
```
I think it should pass if trustedlaunch is selected and type is confidential (higher level settings should always pass when checking for lower levels)